tests/python/test_fields_builder.py

```diff
@@ -3,174 +3,276 @@
 
 import taichi as ti
 
+'''
+Test fields with shape.
+'''
+
 
 @ti.test(arch=[ti.cpu, ti.cuda, ti.vulkan, ti.metal])
 def test_fields_with_shape():
-    n = 5
-    x = ti.field(ti.f32, [n])
 
+    # Initialize common variables for tests below
+    shape_size_1d = 5
+    x = ti.field(ti.f32, [shape_size_1d])
+    y = ti.field(ti.f32, [shape_size_1d])
+
+    # [shape] 1. Test and validation for assign once kernel function
     @ti.kernel
-    def func():
-        for i in range(n):
+    def assign_once_kernel_func():
+        for i in range(shape_size_1d):
             x[i] = i
 
-    func()
-
-    for i in range(n):
+    assign_once_kernel_func()
+    for i in range(shape_size_1d):
         assert x[i] == i
 
-    y = ti.field(ti.f32, [n])
-
+    # [shape] 2. Test and validation for assign twice kernel function
     @ti.kernel
-    def func2():
-        for i in range(n):
+    def assign_twice_kernel_func():
+        for i in range(shape_size_1d):
             y[i] = i * 2
-        for i in range(n):
+        for i in range(shape_size_1d):
             x[i] = i * 3
 
-    func2()
-
-    for i in range(n):
+    assign_twice_kernel_func()
+    for i in range(shape_size_1d):
         assert x[i] == i * 3
         assert y[i] == i * 2
 
-    func()
-
-    for i in range(n):
+    # [shape] 3. Test and validation for Re-assign variable of field
+    assign_once_kernel_func()
+    for i in range(shape_size_1d):
         assert x[i] == i
 
 
+'''
+Test fields with builder dense.
+'''
+
+
 @ti.test(arch=[ti.cpu, ti.cuda, ti.vulkan, ti.metal])
 def test_fields_builder_dense():
-    n = 5
 
+    # Initialize common variables for tests below
```

Comment:
I think the comment in this line is unnecessary

Reply:
Fixed